Cargo.toml

```diff
@@ -43,7 +43,7 @@ release_max_level_debug = []
 release_max_level_trace = []
 
 [dev-dependencies]
-slog-term = "2"
+slog-term = { git = 'https://github.com/valarauca/term', branch = 'opaque_keys'}
```

Comment:
It's easier to use `.cargo/config` file and `paths` section to override it locally.